src/js/ia/Helpers.js

```diff
@@ -129,6 +129,16 @@
         }
     });
 
+    // Check if val1 matches val2 using a regex
+    Handlebars.registerHelper('match', function(val1, val2, options) {
+       val2 = new RegExp(val2);
```

Comment:
Isn't it always just the string 'success' if it's successful? Maybe just do `===`?